superset/databases/filters.py

```diff
@@ -51,3 +57,37 @@ def apply(self, query: Query, value: Any) -> Query:
                 ),
             )
         )
+
+
+class DatabaseUploadEnabledFilter(BaseFilter):  # pylint: disable=too-few-public-methods
+    """
+    Custom filter for the GET list that filters all certified charts
+    """
+
+    name = _("Upload Enabled")
+    arg_name = "upload_is_enabled"
+    model = Database
+
+    def apply(self, query: Query, value: any) -> Query:
+        extra_allowed_databases = []
+        if hasattr(g, "user"):
+            extra_allowed_databases += app.config["ALLOWED_USER_CSV_SCHEMA_FUNC"](
+                Database, g.user
+            )
+        if len(extra_allowed_databases):
```

Comment:
looking at this now, we may want to add a comment that to keep this simple, we're not going to check to see if these schemas actually exist in any of the databases.

Reply:
Added.